libcontainerd/remote/client.go

```diff
@@ -773,14 +773,18 @@ func (c *client) processEventStream(ctx context.Context, ns string) {
 			if err != nil {
 				errStatus, ok := status.FromError(err)
 				if !ok || errStatus.Code() != codes.Canceled {
-					c.logger.WithError(err).Error("Failed to get event")
-					c.logger.Info("Waiting for containerd to be ready to restart event processing")
+					c.logger.WithError(err).Error("failed to get event")
+					c.logger.Info("waiting for containerd to be ready to restart event processing")
 					if c.waitServe(ctx) {
 						go c.processEventStream(ctx, ns)
 						return
 					}
 				}
-				c.logger.WithError(ctx.Err()).Info("stopping event stream following graceful shutdown")
+				if err := ctx.Err(); err != nil {
+					c.logger.WithError(err).Info("stopping event stream following graceful shutdown")
+				} else {
+					c.logger.Info("stopping event stream following graceful shutdown")
+				}
```

Comment:
Code was last changed in  https://github.com/moby/moby/commit/1acaf2aabe000c6101501af321969df7c0ca5413 (https://github.com/moby/moby/pull/39513) wondering if it should be removed altogether, but perhaps there's still errors we want to log ./cc @cpuguy83